### DISCUSSION TITLE
Is there a reason that zoom is not an auto_updates cask?

### DISCUSSION THREAD

**Discussion by alexreg** [Homebrew/discussions#685] [Casks]
The issue is that every time zoom updates, my settings gets erased, which is a real pain. The issues with Zoom's shoddy installer are well-known, but shouldn't we let it auto-update in any case?

**Comment by carlocab** [comment]
From the [`zoom.rb`](https://github.com/Homebrew/homebrew-cask/blob/7442c1bf19ef9df528e1593661002328c7d4a22f/Casks/zoom.rb#L25-L26):

```
  # Do not add `auto_updates`. While supporting an auto-update mechanism, this software is more inconvenient than most
  # See https://github.com/Homebrew/homebrew-cask/pull/93083
```

https://github.com/Homebrew/homebrew-cask/pull/93083

**Reply by alexreg** [reply]
Ah, I see. What a pain that it resets all user settings though! They've made a complete mess of their macOS installer.

**Reply by carlocab** [reply]
It hasn't reset mine... but I think if you'd rather use Zoom's autoupdate functionality rather than `brew upgrade`, you could try `brew pin zoom` so that `brew` doesn't touch your Zoom installation. It'll then be up to you to update it yourself.

**Comment by alexreg** [comment]
Ah, maybe because I’m using brewdo… Thanks for the tip, in any case.

**Comment by alexreg** [comment]
Ah, it seems like casks aren’t pinnable. Is that an oversight?

```
Error: No available formula with the name "zoom".
```

**Reply by carlocab** [reply]
Huh. Weird. I could've sworn you could `pin` casks. But I'm seeing the same thing you are. Whoops; my mistake.

**Reply by alexreg** [reply]
No worries. Do you want to submit an issue for that functionality? :-)
